core/src/utils/test/ready.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { componentOnReady } from '../helpers';
+
+describe('componentOnReady()', () => {
+  it('should correctly call callback for a custom element', async (done) => {
+    customElements.define('hello-world', class extends HTMLElement {
+      constructor() {
```

Comment:
Is this empty constructor needed ?

Reply:
Yeah when you extend HTMLElement you need to call `super()`, so the constructor is needed.

Reply:
Thanks.
I meant to double check but I forgot. TIL.